apps/user-profile/src/app/components/header/header.component.html

```diff
@@ -7,6 +7,16 @@
   <a class="ml-3" [innerHTML]="logo" [routerLink]="disableLogo ? [] : ['/']" queryParamsHandling="merge"></a>
 
   <p [ngStyle]="{'color': textColor}" class="ml-3 mt-auto mb-auto">{{label}}</p>
-</mat-toolbar>
-
 
+  <div id="nav-menu-user-info" class="justify-content-end">
+    <perun-web-apps-header-menu
+      [activeLink]="activeLink"
```

Comment:
do we need this property to be sent into the header menu? isnt it redundant?

Reply:
It's true. I removed this property from this html element and also from the header-menu comomponent.